main.cpp

```diff
@@ -1,475 +1,581 @@
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) name (std::string)
+    2) years experience (int)
+    3) trophies won (int)
+    4) salary (double)
+    5) inspiring (bool)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) give motivational speech
+    2) lose job
+    3) give press conference
+*/
+
+struct Coach
+{
+    //name (std::string)
+    std::string name;
+    //years experience (int)
+    int yearsExperience = 4;
+    //trophies won (int)
+    int numTrophiesWon = 2;
+    //salary (double)
+    double salary = 8078900.99;
+    //inspiring (bool)
+    bool inspiringn = false;
+
+    //give motivational speech
+    void deliverMotivationalSpeech(bool winningStreak);
+    //lose job
+    void getSacked(int numberOfGamesInLosingStreak);
+    //give press conference
+    void givePressConference(bool wonGame);
+};
 
 /*
-Thing 9)
+Thing 7) Fan
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) name (std::string)
+    2) age (int)
+    3) season ticket holder (bool)
+    4) distance from stadium (double)
+    5) fanclub member (bool)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) buy ticket to game
+    2) cheer
+    3) wear team colours
+*/
+
+struct Fan
+{
+    //name (std::string)
+    std::string name = "John Johnson";
+    //age (int)
+    int age;
+    //season ticket holder (bool)
+    bool hasSeasonTicket = true;
+    //distance from stadium (double)
+    double distanceFromStadiumInKM = 345.78;
+    //fanclub member (bool)
+    bool fanclubMember = true;
+
+    //buy ticket to game
+    void purchseTicket(float ticketPrice, std::string opponentTeam);
+    //cheer
+    void cheer(bool teamScoredGoal);
+    //wear team colours
+    void wearTeamColours(bool teamWinningStreak);
+};
 
 /*
-You've just defined 10 UDTs!
-4 of them are unrelated (UDTs 1-4).
-5 of them form the sub-objects that make up the 10th UDT. 
+Thing 8) Medic
+5 properties:
+    1) qualifications (std::string)
+    2) name (std::string)
+    3) salary (double)
+    4) explayer (bool)
+    5) IT skills (bool)
+3 things it can do:
+    1) give steriod injection
+    2) massage treatment
+    3) make fitness assessment
+*/
 
-Move them below this block comment and put them in numerical order 
-(1 - 10).
+struct Medic
+{
+    //qualifications (std::string)
+    std::string qualifications;
+    //name (std::string)
+    std::string name = "The FA Level 4 Emergency Medical Management in Football";
+    //salary (double)
+    double salary = 178909.99;
+    //explayer (bool)
+    bool isExPlayer = true;
+    //IT skills (bool)
+    bool hasITSkills = true;
+
+    //give steriod injection
+    void giveSteriodInjection(Player player, std::string bodyPart, bool servePain);
+    //massage treatment
+    void giveMassageTreatment(Player player, std::string therapyTargetArea);
+    //make fitness assessment
+    float getFitnessAssessmentScore(int restingBPM, int maxBPM, float bloodPressure); //returns value of a players fitness assessment
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1
`giveFitnessAssessment()` or `provideFitnessAssessment()` would work.
- [ ] revise